src/AppBundle/Resources/views/_partials/Delivery/actions.html.twig

```diff
@@ -0,0 +1,25 @@
+{% if is_granted('ROLE_ADMIN') and delivery.status == 'WAITING' %}
+  <button type="button" class="btn btn-xs btn-success"
+    data-toggle="modal"
+    data-target="#delivery-modal"
+    data-delivery="{{ delivery.id }}"
+    data-action="dispatch">
+    {% trans %}delivery.status.DISPATCHED.verb{% endtrans %}
+  </button>
+{% elseif is_granted('ROLE_COURIER') and delivery.status == 'DISPATCHED' %}
```

Comment:
I think you should also allow admin to do this.

Use-case: an admin is checking all the deliveries at the end of the day and need to update the statuses

Reply:
Actually `ROLE_ADMIN` inherits from `ROLE_COURIER` 😉 

https://github.com/coopcycle/coopcycle-web/blob/b05e11e7cc14f748a10e3eecd9efc8d8a94a8faa/app/config/security.yml#L13